src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -24,6 +24,15 @@
 
 extern int kds_mode;
 
+static int zocl_load_aie_only_pdi(struct drm_zocl_dev *zdev, struct axlf *axlf,
+			char __user *xclbin, struct sched_client_ctx *client);
```

Comment:
We can move the function a head of zocl_xclbin_load_pdi so that we don't need to declare it here. This seems to be convention of this implementation.